src/views/blocks/ImageTextBlock.php

```diff
@@ -4,26 +4,31 @@
 /**
  * @var $this \luya\cms\base\PhpBlockView
  */
+
+$image = $this->extraValue('image');
+$text = $this->extraValue('text');
 ?>
-<?php if ($this->extraValue('image') && $this->extraValue('text')): $imageCaption = $this->extraValue('image')['caption'];?>
+<?php if ($image && $text): ?>
+    <?php $imageCaption = $image['caption']; ?>
+
     <div class="media">
         <div class="<?php echo ($this->varValue('imagePosition', 'left') == 'left') ? 'media-left pull-left' : 'media-right pull-right' ?>">
-            <?= Html::img($this->extraValue('image')['source'], [
+            <?= Html::img($image['source'], [
                 'class' => 'media-object',
                 'alt' => $imageCaption === null ? '' : $imageCaption,
                 'title' => $imageCaption,
                 'width' => $this->cfgValue('width', null),
                 'height' => $this->cfgValue('height', null),
-                'style' => (($this->varValue('imagePosition', 'left') == 'left') ? "margin-right:{$this->cfgValue('margin', '20px')}" : "margin-left:{$this->cfgValue('margin', '20px')}") . $this->cfgValue('margin', '20px', ';margin-bottom:{{margin}};'),
+                'style' => $this->extraValue('imagePosition') . $this->cfgValue('margin', '20px', ';margin-bottom:{{margin}};'),
```

Comment:
@boehsermoe e.g. here

Reply:
The `20px` is just the default value. Do u think it is better to make a constant for it?

Reply:
I thought there is a class like `m-l-2` like in the BS4 others but there is not.. 
maybe just  leave it like that, sry  didin´t used BS3 for a long time.

In general, it´s always good to avoid hardcoded values, but your PR makes better than before .. thank you. for me it´s fine from FE side